### ISSUE TITLE
User not in sensitive data group can inadvertently change Approved state of member account

### ISSUE THREAD

**Issue by russellshome** [umbraco/Umbraco-CMS#16873]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.8.6

### Bug summary

A user that is not in the sensitive data group should not be able to change the approved state of a member account. However, when a user that is not in the sensitive data group saves a member, that member is set to not approved.

### Specifics

This happens in version 10 but does not happen in version 13 of Umbraco

### Steps to reproduce

Create Umbraco 10 site. 
Add a member using the default admin account. The member is approved by default.
Change the Admin user account by removing the sensitive data group
Go to the member and Save
Change the Admin user account by adding back the sensitive data group
Check whether the member is approved. It will have become not approved.

### Expected result / actual result

Expected: User not in sensitive data group cannot change approved state of a member
Actual: User not in sensitive data group sets approved to false on save

**Comment by elit0451** [comment]
Hi @russellshome 👋 

Thanks for reaching out! I was able to reproduce this issue when a member is saved from a user who is **not** part of the Sensitive data user group, the "Approved" toggle is automatically switched off. Unfortunately, since Umbraco 10 is no longer in its support phase, the only thing we can do here is to advise you to upgrade to version 13, where as you pointed out, the problem was resolved.

Here is the list of **Long-term Support & End-of-Life for Umbraco CMS** versions for reference: https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/
